horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
i like too the tabular-like style. I think its eaysier for reading but all other changes could be merged.